JavaScriptSample-subtlecrypto.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div></div>
+
+    <script>
+      (async () => {
+        const publicKey = "xxx"; //add the public encryption key here
+        const TO_ENCRYPT = "hotdog_is_a_s@ndwich.com"; // this is what we are encrypting
```

Comment:
why is `publicKey` camel case and `TO_ENCRYPT` upper snake case?

Reply:
🤦 yes will change casing to be uniform